packages/usa-button/src/styles/_usa-button.scss

```diff
@@ -52,6 +52,13 @@ $button-stroke: inset 0 0 0 units($theme-button-stroke-width);
     @include button-disabled;
   }
 
+  // @TODO: Use `:has()` selector once browser support improves.
+  // https://developer.mozilla.org/en-US/docs/Web/CSS/:has#browser_compatibility
+  .usa-icon {
+    margin-top: -0.1em; // Similar pattern to _usa-breadcrumb.scss:14.
+    vertical-align: middle;
+  }
+
```

Comment:
@amyleadem 

Re: display: `inline-flex`. Because this is such a widely used button ― I wanted to minimize any potentially breaking changes as much as possible. 

Since we already specify `inline-block`, I don't think the switch to `inline-flex` is as drastic as I imagined. Double-checking MDN [display - CSS: Cascading Style Sheets | MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/display#grouped_values) and there should be very little display difference at all.

This addition improves the experience, but might require a new theme setting for the column gap. I'll play with this.